unison-src/transcripts/type-deps.md

```diff
@@ -0,0 +1,31 @@
+# Ensure type dependencies are properly considered in slurping
```

Comment:
This transcript fails on trunk with 

```
⚙️   Processing stanza 8 of 8.ucm: UnknownDependency "dhf3hodo5mv220et7mt6g14e69e1phleffl411te6hf31q81niqt2maui806tj69j04ap796lgmu3isg7brevotlhcb2m9van262e0g"
CallStack (from HasCallStack):
  error, called at src/Unison/Codebase/SqliteCodebase.hs:956:29 in unison-parser-typechecker-0.0.0-He2Hp1llokT2nN4MnUfUXz:Unison.Codebase.SqliteCodebase
```

Because `Z` 's blockage by `Y` isn't detected.